src/coverlet.core/CoverageResult.cs

```diff
@@ -111,7 +111,7 @@ internal void Merge(Modules modules)
             }
         }
 
-        public ThresholdTypeFlags GetThresholdTypesBelowThreshold(CoverageSummary summary, double threshold, ThresholdTypeFlags thresholdTypes, ThresholdStatistic thresholdStat)
+        public ThresholdTypeFlags GetThresholdTypesBelowThreshold(CoverageSummary summary, Dictionary<ThresholdTypeFlags, double> thresholdTypeFlagValues, ThresholdTypeFlags thresholdTypes, ThresholdStatistic thresholdStat)
```

Comment:
I guess we now could remove the ```thresholdTypes``` parameter from the method. The information it provided is now in the ```thresholdTypeFlagValues``` dictionary?

Reply:
makes sense. removed `thresholdTypes` please check

Reply:
Thanks